spec/cocaine/command_line_spec.rb

```diff
@@ -126,6 +126,12 @@
     cmd.command.should == "convert 'a.jpg' xc:black 'b.jpg'"
   end
 
+  it 'handles symbols in user supplied values' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.